cabal.project

```diff
@@ -330,6 +330,12 @@ source-repository-package
   tag: ee59880f47ab835dbd73bea0847dab7869fc20d8
   --sha256: 1lrzknw765pz2j97nvv9ip3l1mcpf2zr4n56hwlz0rk7wq7ls4cm
 
+source-repository-package
+  type: git
+  location: https://github.com/haskell-works/hw-aeson
```

Comment:
This library is already a dependency via `hedgehog-extras`.  The inclusion here is just because the index state hasn't been updated yet so the latest required version is not available.